hey.go

```diff
@@ -101,9 +107,15 @@ Options:
   -disable-redirects    Disable following of HTTP redirects
   -cpus                 Number of used cpu cores.
                         (default for current machine is %d cores)
+
+  -r  If specified, the HTTP request will be randomly selected from the ones passed
+      via -d or -D argument.
 `
 
 func main() {
+	// Specify seed for random generator
+	rand.Seed(time.Now().UnixNano())
```

Comment:
`rand.Seed()` is deprecated, you can see that here: [rand package](https://pkg.go.dev/math/rand@master#Seed) instead you have to use `rand.New()`